config_defaults_inc.php

```diff
@@ -4206,13 +4206,12 @@
  * Recommended config_inc.php settings for developers (these are automatically
  * set if the server is localhost):
  * $g_display_errors = array(
- *     E_RECOVERABLE_ERROR => DISPLAY_ERROR_HALT,
```

Comment:
@dregad I suppose it's not intended to remove this line.

Reply:
Yes it is actually. E_RECOVERABLE_ERROR Is forced to _halt_ by our error handler [[1]] so this is redundant 
[1]: https://github.com/dregad/mantisbt/blob/a212dc04c3027d3b0839ab3513a8427004e00cf3/core/error_api.php#L182

Reply:
I am confused as you removed E_RECOVERABLE_ERROR from comment but added it in code for localhost.

Reply:
Thanks for catching the mistake. I'll fix it.